package.json

```diff
@@ -12,11 +12,16 @@
   "author": "Sabeel Khan",
   "license": "ISC",
   "dependencies": {
+    "cloudinary": "^1.39.0",
     "dotenv": "^16.3.1",
     "ejs": "^3.1.9",
     "ejs-mate": "^4.0.0",
     "express": "^4.18.2",
-    "mongoose": "^7.3.4"
+    "express-fileupload": "^1.4.0",
+    "formidable": "^3.5.0",
+    "mongoose": "^7.3.4",
+    "multer": "^1.4.5-lts.1",
+    "multer-storage-cloudinary": "^4.0.0"
```

Comment:
Same applies to these dependencies as, remove it

Reply:
Removed these as well.